src/eq_schema/Block.js

```diff
@@ -1,14 +1,21 @@
 const Question = require("./Question");
 const RoutingRule = require("./RoutingRule");
 const RoutingDestination = require("./RoutingDestination");
-const { get, isNil, remove, isEmpty } = require("lodash");
+const { get, isNil, remove, isEmpty, map, last, find } = require("lodash");
 const { getInnerHTML } = require("../utils/HTMLUtils");
 
 const pageTypeMappings = {
   QuestionPage: "Question",
   InterstitialPage: "Interstitial"
 };
 
+const isLastPageInSection = (page, ctx) => {
+  const sections = get(ctx, "sections", []);
+  const lastPages = map(sections, section => last(get(section, "pages", [])));
```

Comment:
I feel like there's some FP solution to this but damned if I know what it is.

Reply:
I _think_ something like this would work (all `lodash/fp`)

```js
const getLastPage = pipe(
  getOr([], "pages"),
  last
);

const isLastPageInSection = (page, ctx) =>
  pipe(
    getOr([], "sections"),
    map(getLastPage),
    any({ id: page.id })
  )(ctx);
```

Though, you should be able to safely assume `section` and `pages` will be defined

Reply:
@samiwel be aware, `any` is only available in `lodash/fp`. It's an alias of `some` from regular lodash